File: app/src/main/java/bogomolov/aa/fitrack/repository/entities/PointEntity.java
package bogomolov.aa.fitrack.repository.entities;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;


@Entity
public class PointEntity {
    @PrimaryKey(autoGenerate = true)
    private long id;
    private long time;
    private double lat;
    private double lng;
    private int smoothed;

    public int getSmoothed() {
        return smoothed;
    }

    public void setTime(long time) {
        this.time = time;
    }

    public void setSmoothed(int smoothed) {
        this.smoothed = smoothed;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getId() {
        return id;
    }

    public long getTime() {
        return time;
    }

    public double getLat() {
        return lat;
    }

    public double getLng() {
        return lng;
    }

    public void setLat(double lat) {
        this.lat = lat;
    }

    public void setLng(double lng) {
        this.lng = lng;
    }
}

File: app/src/main/java/bogomolov/aa/fitrack/repository/entities/PointEntity.kt
package bogomolov.aa.fitrack.repository.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class PointEntity (
    @PrimaryKey(autoGenerate = true)
    var id: Long = 0,
    var time: Long = 0,
    var lat: Double = 0.0,
    var lng: Double = 0.0,
    var smoothed: Int = 0
)